### PR TITLE
Add gaze17 (17" 3060)

### PR DESCRIPTION
**Pull request by jacobgkau**
This add the Gazelle 17 to tech docs. This only adds the 17" 3060 version since no other versions have currently been tested or listed for sale; a follow-up PR will need to add the specs and photos for the other versions when they're available.

I'd still like to swap out the product photo for the one being used on the sales page (with the display powered on and showing the Coreboot logo), but aside from that, this should be ready for proofreading.

### CODE REVIEW

File: src/models/gaze17/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- Memory
+    - Up to 64GB (2x32GB) dual-channel DDR4 SO-DIMMs @ 3200 MHz
+- Networking
+    - Gigabit Ethernet
+        - Realtek RTL8111H
+    - M.2 PCIe/CNVi WiFi/Bluetooth
+        - Intel Wi-Fi 6 AX200/AX201
+- Power
+    - 180W DC-in port
+        - Barrel size: 5.5mm (outer), 2.5mm (innter)
```
Comment (leviport) [inline src/models/gaze17/README.md:40]: s/innter/inner

File: src/models/gaze17/external-overview.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+|![Fn-F8](./img/fn-f8.png)  |Fn+F8   |Screen brightness down       |
+|![Fn-F9](./img/fn-f9.png)  |Fn+F9   |Screen brightness up         |
+|![Fn-F10](./img/fn-f10.png)|Fn+F10  |Toggle webcam                |
+|![Fn-F11](./img/fn-f11.png)|Fn+F11  |Toggle airplane mode         |
+|![Fn-F12](./img/fn-f12.png)|Fn+F12  |Suspend                      |
+|![Fn-F12](./img/fn-dia.jpg)|Fn+`    |Play/Pause                   |
+|![Fn-*](./img/fn-star.png) |Fn+*    |Toggle keyboard backlight    |
+|/                          |Fn+/    |Cycle keyboard color         |
+|-                          |Fn+-    |Decrease keyboard brightness |
+|+                          |Fn++    |Increase keyboard brightness |
```
Comment (leviport) [inline src/models/gaze17/external-overview.md:63]: Should we include `Fn`+`1` for fan toggle? Looks like we aren't super consistent about it, but it's probably good to include.

File: src/models/gaze17/repairs.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the 13 bottom panel screws.
+
+![Bottom panel screws](./img/bottom-panel-screws.webp)
+
+3. Pull the bottom panel off, starting from the hinges in the back.
+
+## Replacing the RAM:
+
+The Gazelle 16 supports up to 64GB (2x32GB) of DDR4 SO-DIMMs running at 3200MHz. If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
```
Comment (leviport) [inline src/models/gaze17/repairs.md:34]: s/Gazelle 16/Gazelle 17

File: src/models/gaze17/repairs.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+### Steps to replace the CMOS battery:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Follow the steps above to [remove the outermost M.2 drive](#replacing-an-m2nvme-ssd), then pull the battery away from the adhesive holding it to the case. 
+    - If you are not replacing the CMOS battery, this step is optional, but will make it easier to unplug the CMOS battery connector.
+3. Unplug the white connector that connects the CMOS battery to the motherboard.
+
+![CMOS battery](./img/cmos-battery.webp)
+
+4. To clear the CMOS, [disconnect the main battery](replacing-the-battery), open the lid of the machine, and hold down the power button for at least 15 seconds to discharge any residual energy in the system.
```
Comment (leviport) [inline src/models/gaze17/repairs.md:125]: Link is broke, I think it needs a `#` before it

File: src/models/gaze17/repairs.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+5. Re-connect the CMOS battery.
+6. [Reconnect the main battery](replacing-the-battery), [replace the M.2 SSD](#replacing-an-m2nvme-ssd), and [replace the bottom cover](#removing-the-bottom-cover).
```
Comment (leviport) [inline src/models/gaze17/repairs.md:127]: Same link issue as above

File: src/models/gaze17/repairs.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+9. Carefully replace the heatsink.
+10. Replace the silver heatsink screws, starting with #1, then #2, and so on until #6.
+11. Replace the silver fan screws.
+12. Plug the two white fan connectors back into the motherboard.
+
+## Replacing the keyboard:
+
+Keyboard replacement is simple and requires only a cross-head screwdriver.
+
+**Tools required:** Cross-head (Phillips) screwdriver
```
Comment (leviport) [inline src/models/gaze17/repairs.md:169]: Should we mention tweezers as optionally requried in "Tools required" and in the sentence above it? Is a screwdriver even required, or is it just adhesive holding the thing in?

File: src/models/gaze17/repairs.md [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+![Keyboard ribbon connectors](./img/keyboard-ribbons.webp)
+
+7. Flip the black latches upwards to free the ribbon cables.
+8. Pull the ribbon cables out of the connectors.
+9. Remove the keyboard and replace it with the new one.
+10. Carefully slide both ribbon cables into their connectors.
+11. Flip the black latches back into place to secure the ribbon cables.
+12. Place the keyboard back into position, starting with the tabs on the bottom edge.
+13. Secure the rest of the keyboard by pressing down on each of its edges.
+14. Turn the machine lid-side down again and replace the two keyboard screws.
```
Comment (leviport) [inline src/models/gaze17/repairs.md:205]: `replace the two keyboard screws` should probably be `[replace the bottom cover](#removing-the-bottom-cover.)`

**Comment by leviport** [comment]
I noticed a few small things, but this otherwise looks great!

File: src/models/gaze17/README.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+            - LCD panel: AUO B173HAN04.9 (or equivalent)
+    - External video output:
+        - 1x HDMI
+        - 1x Mini DisplayPort 1.2
+        - 1x DisplayPort 1.4 over USB-C
+- Memory
+    - Up to 64GB (2x32GB) dual-channel DDR4 SO-DIMMs @ 3200 MHz
+- Networking
+    - Gigabit Ethernet
+        - Realtek RTL8111H
```
Comment (thomas-zimmerman) [inline src/models/gaze17/README.md:35]: Would be good to link to the chip page: https://www.realtek.com/en/products/communications-network-ics/item/rtl8111h-s-cg

File: src/models/gaze17/internal-overview.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+## Bottom components and connectors:
+
+![Internal Components](./img/components-highlighted.webp)
+
+- Webcam/microphone connector is highlighted in pink
+- Fan connectors are highlighted in orange
+- LCD panel connector is highlighted in green
+- Wireless card is highlighted in blue
+- RAM is highlighted in red
+- M.2 SSDs are highlighted in yellow
```
Comment (thomas-zimmerman) [inline src/models/gaze17/internal-overview.md:12]: The drive slots have different spec support for NVMe gen 3/SATA and just NVMe gen4 support and should be called out on this overview

File: src/models/gaze17/repairs.md [hunk 10]
```diff
@@ -0,0 +1,205 @@
...
+
+## Replacing the CMOS battery:
+
+The CMOS battery supplies power to the system's CMOS chip. UEFI settings and the computer's hardware clock are stored on the CMOS. If your system doesn't boot, you can reset the CMOS to force a low-level hardware reset. If your clock is constantly resetting, it's likely your CMOS battery needs to be replaced.
+
+**Tools required:** Cross-head (Phillips) screwdriver    
+**Time estimate:** 15 minutes    
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the CMOS battery:
```
Comment (thomas-zimmerman) [inline src/models/gaze17/repairs.md:116]: Is there a documented plug type if Cx needs to replace the 2-wire CMOS battery? Online there seems to be ~3 types of connectors that won't all fit into the MB

**Review by thomas-zimmerman — COMMENTED** [review]
Very nice first pass.

File: src/models/gaze17/repairs.md (hunk 10 above)
Comment (jacobgkau) [reply src/models/gaze17/repairs.md:116]: No there is not.

File: src/models/gaze17/README.md (hunk 1 above)
Comment (jacobgkau) [reply src/models/gaze17/README.md:40]: Fixed with 561e43a115448abf3b41ce7771f51c8bd6c88724.

File: src/models/gaze17/external-overview.md (hunk 2 above)
Comment (jacobgkau) [reply src/models/gaze17/external-overview.md:63]: Yes, we should include that. Added with d8caecae30da0950f47cea6deb8afabe73767012.

File: src/models/gaze17/repairs.md (hunk 3 above)
Comment (jacobgkau) [reply src/models/gaze17/repairs.md:34]: Of course. Fixed with dabd0ecbacdd840e8dcfe1f78d7d9e4262653a94.

File: src/models/gaze17/repairs.md (hunk 5 above)
Comment (jacobgkau) [reply src/models/gaze17/repairs.md:127]: Fixed with dabd0ecbacdd840e8dcfe1f78d7d9e4262653a94.

File: src/models/gaze17/repairs.md (hunk 6 above)
Comment (jacobgkau) [reply src/models/gaze17/repairs.md]: Yes, I've copied the verbiage from the oryp8 page over with df22a2805443e1e1abfa9f9df4d59b6aa03ff100.

File: src/models/gaze17/repairs.md (hunk 7 above)
Comment (jacobgkau) [reply src/models/gaze17/repairs.md]: Fixed with df22a2805443e1e1abfa9f9df4d59b6aa03ff100.

File: src/models/gaze17/internal-overview.md (hunk 9 above)
Comment (jacobgkau) [reply src/models/gaze17/internal-overview.md:12]: We have already had this conversation. https://github.com/system76/tech-docs/pull/87#discussion_r676792485

File: src/models/gaze17/README.md (hunk 8 above)
Comment (jacobgkau) [reply src/models/gaze17/README.md:35]: Done with 3f75f87a5a7985bc08261c40b4b3d236ef95f278.

File: src/models/gaze17/internal-overview.md (hunk 9 above)
Comment (jacobgkau) [reply src/models/gaze17/internal-overview.md:12]: Addressed with 80a96d06d75277efaecd488944d89559edc2c7ba even though I disagree with the need for this.

**Pull request merged**